common/xml/convert_and_merge_composable_tests/composable-loc-implicit-parent.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<!--
+  Convert
+    <loc ...><port ...><port ...></loc>
+  to
+    <loc ...>BLOCK.PORT BLOCK.PORT</loc>
+  -->
+<xml>
+ <pb_type name="parent">
+  <input name="i" width="2" />
+  <loc>
```

Comment:
Anyway to limit this to ``<pinlocations><loc>``?

Reply:
Yes -- but are `<loc>` used anywhere else?

Reply:
I did it anyway....